apisix/plugins/error-log-logger.lua

```diff
@@ -214,6 +227,70 @@ local function send_to_skywalking(log_message)
 end
 
 
+local function send_to_clickhouse(log_message)
+    local err_msg
+    local res = true
+    local url_decoded = url.parse(config.clickhouse.endpoint_addr)
```

Comment:
we can use `httpc:request_uri` directly like the skywalking destination?

Reply:
yes